packages/shared/src/utils/files/file-uploader-utils.tsx

```diff
@@ -44,7 +81,10 @@ export const readFiles = (files, getFileReadErrorMessage, settings) => {
                     documentType: settings?.documentType || 'utility_bill',
                     documentFormat: getFormatFromMIME(f),
                     file_size: f.size,
-                    ...settings,
```

Comment:
@niloo-fs why can't we use `...settings` here? 🙂

Reply:
because of this:
**'documentType' is specified more than once, so this usage will be overwritten.ts(2783)
file-uploader-utils.tsx(82, 21): This spread always overwrites this property.**

Reply:
@niloo-fs i see) what do you think about moving `documentType` to the end 🙂?
                    ```
                    ...settings,
                    documentType: settings?.documentType || 'utility_bill',
```